src/bootstrap/dist.rs

```diff
@@ -1186,7 +1186,7 @@ pub fn sanitize_sh(path: &Path) -> String {
     return change_drive(unc_to_lfs(&path)).unwrap_or(path);
 
     fn unc_to_lfs(s: &str) -> &str {
-        if s.starts_with("//?/") { &s[4..] } else { s }
+        if let Some(stripped) = s.strip_prefix("//?/") { stripped } else { s }
```

Comment:
Sorry to barge in :)

This could even more simply be:
```suggestion
        s.strip_prefix("//?/").unwrap_or_else(s)
```

Reply:
updated, thanks!